clients/geth/specular/rollup/services/validator/validator.go

```diff
@@ -24,6 +25,14 @@ type challengeCtx struct {
 	lastValidatedAssertion *rollupTypes.Assertion
 }
 
+type CustomError struct {
+	Message string
+	Code int
```

Comment:
i'd avoid adding this for this PR until we have a concrete plan for exactly how to use it (what codes to assign to what types of errors)

Reply:
CustomError are removed for now